components/inputs/input-time.js

```diff
@@ -1,12 +1,29 @@
+import '../dropdown/dropdown.js';
+import '../dropdown/dropdown-menu.js';
+import '../menu/menu.js';
+import '../menu/menu-item-radio.js';
+
 import { css, html, LitElement } from 'lit-element/lit-element.js';
 import { formatTime, parseTime } from '@brightspace-ui/intl/lib/dateTime.js';
+import { bodySmallStyles } from '../typography/styles.js';
 import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { inputLabelStyles } from './input-label-styles.js';
 import { inputStyles } from './input-styles.js';
 
 const VALUE_RE = /^([0-9]{1,2}):([0-9]{1,2})(:([0-9]{1,2}))?$/;
 const TODAY = new Date();
 const DEFAULT_VALUE = new Date(TODAY.getFullYear(), TODAY.getMonth(), TODAY.getDate(), 0, 0, 0);
+const INTERVALS = getIntervals();
```

Comment:
So this code is going to run immediately in the browser when the script is included -- whether or not a time input is actually rendered on the page. Therefore, it might be better to just call `getIntervals()` directly in the `render()` method. It'll run each time the component is rendered, but won't run when it's not on the page.

Another option to get the best of both worlds would be something like:
```
let intervals = null;
function getIntervals() {
    if (intervals !== null) return intervals;
    ...
}
```